src/course-outline/hooks.jsx

```diff
@@ -27,15 +42,35 @@ const useCourseOutline = ({ courseId }) => {
     },
   };
 
+  const handleEnableHighlightsSubmit = () => {
+    setIsQueryPending(true);
```

Comment:
instead of this use `dispatch(updateSavingStatus({ status: RequestStatus.PENDING }));`

Reply:
Done